tests/metagpt/actions/test_sd_design.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# @Date    : 2023/7/22 02:40
+# @Author  : stellahong (stellahong@fuzhi.ai)
+#
+
+import pytest
+from typing import List
+from metagpt.actions.design import SDPromptOptimize, SDPromptImprove
+from metagpt.actions.ui_design import ModelSelection
+
+@pytest.mark.asyncio
+async def test_ui_model_selection():
+    ms = ModelSelection()
+    model_name, domain = await ms.run("Pokémon Go")
+    assert model_name=="pixelmix_v10"
+    
+    
+@pytest.mark.asyncio
+async def test_ui_sd_generation():
+    pass
+
+@pytest.mark.asyncio
+async def test_ui_sd_prompt_optimize():
+    sd_po = SDPromptOptimize()
+    resp = await sd_po.run(query="Pokémon Go", domain="Anime", answer_count=1)
```

Comment:
Usually, `answer_count` is an uncertain value. So, suggest to add a test of `answer_count=k(2 or 3)` to make the `run` result be more stable.